src/content/docs/mobile-monitoring/new-relic-mobile-cordova-phonegap/get-started/get-started-with-cordova-monitoring.mdx

```diff
@@ -0,0 +1,123 @@
+  - /docs/mobile-monitoring/new-relic-mobile-cordova-phonegap/get-started/introduction-new-relic-cordova 
+---
+
+import mobileQueryExample from 'images/mobile_screenshot-full_hybrid-summary.png'
+
+The official New Relic Cordova plugin for iOS and Android allows developers to easily embed the mobile agents into [Cordova applications](http://cordova.apache.org/). Written in JavaScript, the plugin automatically includes New Relic's native agents to provide mobile monitoring and performance visibility.
+
+## Features
+
+Once installed, the New Relic Cordova plugin will:
+
+* Detect the platforms added to your Cordova application and apply the most recent release of the relevant New Relic [Android](/docs/releases/android) or [iOS](/docs/releases/ios) agent
+* Add post-build scripts for uploading iOS symbolication files
+* Upload Android Proguard mapping files
+* Automatically instrument mobile applications built via Cordova
+* Capture JavaScript errors
+* Generate distributed traces
+* Track promise rejection (an unhandled exception in JavaScript)
+* Track warnings and errors with console logs
+* Capture interactions and their sequences
+* Track user sessions
+
+## Get started with Cordova monitoring
+
+### Step 1: Check the requirements [#prerequisites]
+
+Before installing, make sure you've met these prerequisites for adding the Android or iOS platform to your Cordova project:
+
+* Cordova 7.x or higher
+* Node.js 6.0 or higher
+* Cordova CLI tools
+* An [Android](https://cordova.apache.org/docs/en/latest/guide/platforms/android) or [iOS](https://cordova.apache.org/docs/en/latest/guide/platforms/ios/index.html) Cordova platform for your project
+* Mobile [application tokens](/docs/mobile-monitoring/new-relic-mobile/maintenance/viewing-your-application-token)
+
+### Step 2: Get your application token
+If you’re new to New Relic, [create an account](https://newrelic.com/signup) - it's free!
+
+First, you'll need to get your application token, which will be used in Step 3 to connect your Cordova app with New Relic:
+
+  1. Go to **one.newrelic.com > Browser & Mobile > (choose Android or iOS)**, then complete the instrumentation steps in the UI. 
+  2. From Step 4 in the UI, copy the application token. You’ll need this when installing the Cordova plugin.
+ 
+  <Callout variant="important">
+  If your app is built on both iOS and Android, you’ll need to get application tokens for iOS and Android separately.
+  </Callout> 
+
+### Step 3: Install the Cordova plugin [#install]
+
+1. In your Cordova project directory, add the Cordova plugin to your project with the Cordova command line tool, and paste the application token(s) (from step 2) using the `--variable` argument:
+
+   ```
+   # Install from github repository
+   cordova plugin add https://github.com/newrelic/newrelic-cordova-plugin.git --variable IOS_APP_TOKEN="{ios-app-token}" --variable ANDROID_APP_TOKEN="{android-app-token}"
+   ```
+
+2. (Optional) If you're using ionic install, add this:
```

Comment:
```suggestion
2. (Optional) If your app uses the ionic cordova build, add this:
```
@ndesai-newrelic , where do they add the code? In their Cordova project directory?